server/pipeline_friend.go

```diff
@@ -100,7 +103,39 @@ func (p *pipeline) addFacebookFriends(logger *zap.Logger, userID []byte, accessT
 				if err != nil {
 					logger.Error("Could not commit transaction", zap.Error(err))
 				} else {
-					logger.Info("Imported friends from Facebook")
+					logger.Debug("Imported friends from Facebook")
+
+					// Send out notifications.
+					if len(friendUserIDs) != 0 {
+						content, err := json.Marshal(map[string]interface{}{"handle": handle, "facebook_id": fbid})
```

Comment:
The sender always contains the User ID anyway, so no need to add it to the content. The only things added to the content are data fields that don't already appear in the top-level notification fields or are embedded in the subject: "foo wants to be your friend" -> content is `{"handle": "foo"}`. Enough information to display a custom/localised notification as needed.